specification/apicenter/ApiCenter.Management/models.tsp

```diff
@@ -266,6 +267,37 @@ model MetadataSchemaProperties {
   assignedTo?: MetadataAssignment[];
 }
 
+@added(Versions.v2024_06_01_preview)
+@doc("The import from Azure API Management request.")
+model ImportFromApimRequest {
+  @doc("An entity the metadata schema is requested for.")
+  sourceResourceIds?: string[];
```

Comment:
>string [](http://example.com/codeflow?start=22&length=6)

If this a fully qualified ARM Id ? if so, please use armResourceIdentifier[] instead of string[]

Reply:
Removed this functionality for now. It'll go to next version.